tmux/tmux.conf.symlink

```diff
@@ -1,8 +1,12 @@
 # Make mouse useful in copy mode
 setw -g mouse on
 
+# Terminal type config
+
+# Help with colors
+set -g default-terminal "xterm-kitty"
```

Comment:
I _think_ I found an Issue on the Kitty repo that suggested this. I should update the comment to link to th eissue.